apps/website/inertia/components/home/highlights/highlights.tsx

```diff
@@ -0,0 +1,79 @@
+import { Link } from "@tuyau/inertia/react";
+import HighlightCard from "./highlight_card";
+
+import { ScrollArea, ScrollBar } from "~/components/ui/scroll-area";
+
+export enum EventType {
+  Workshop = "Workshop",
+  Talk = "Palestra",
+}
+
+export default function Highlights() {
+  return (
+    <div className="w-full">
+      <div className="flex flex-col gap-y-4">
```

Comment:
Nitpick: stylistically this does not make any difference but we should strive as much as possible to make the website accessible: we don't know (yet) who is using the website. This can be achieved by, among other things, using semantically correct markup. My suggestion would be to use a `section` element here (since this marks a different section of the website) and go from there.
Again, just a nitpick since visually everything works out fine.